log/bulk_output.go

```diff
@@ -79,26 +79,34 @@ func NewBulkOutput(writer io.Writer, formatter LogFormatter, bulkSize int) Outpu
 }
 
 type httpWriter struct {
-	url string
+	url     string
+	timeout time.Duration
 }
 
 func (w *httpWriter) Write(p []byte) (n int, err error) {
 	reader := bytes.NewReader(p)
 	size := reader.Len()
-	resp, err := http.Post(w.url, "application/json", reader)
+	client := &http.Client{
```

Comment:
note that previously we re-used a default client that was defined at the `http` package level and now we're allocating a new struct for each HTTP post. Probably not significant but I'm not sure you noticed that you changed the behavior.

Reply:
Yes. This made more sense with the `setTimeout(..)` implementation but now it really doesn't  make much sense. Will fix